docker-compose.yml

```diff
@@ -12,28 +12,31 @@ services:
     environment:
       KAFKA_ADVERTISED_HOST_NAME: ${DOCKER_HOST_IP}
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_CREATE_TOPICS: "test:1:1"
+      KAFKA_CREATE_TOPICS: "topic.OdeBsmJson:1:1"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
 
   s3d:
```

Comment:
Running the s3d through docker-compose seems to result in a deployment with no logging. I tried messing around with the logback.xml and then rebuilding the image to see if it only used the default values but they too seemed to have no effect. Very confused here.

Reply:
I can verify that this is happening. The only output we get from the S3D when running with docker-compose is the following:

> my-jpo-s3-deposit-s3d-1        | log4j:WARN No appenders could be found for logger (us.dot.its.jpo.ode.aws.depositor.AwsDepositor).
my-jpo-s3-deposit-s3d-1        | log4j:WARN Please initialize the log4j system properly.
my-jpo-s3-deposit-s3d-1        | log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.

I'll look into this, but it might be out of the scope of this PR.

Reply:
I've created a new work item for this.